R/initialize_project.R

```diff
@@ -27,7 +29,9 @@ initialize_project <- function(folders = c('scripts', 'original_data',
                                            'results', 'results/quarto_output'),
                                quarto = TRUE,
                                organization = "WDFW",
-                               renv = FALSE){
+                               renv = FALSE,
+                               overwrite = TRUE,
```

Comment:
might want to rename this to something more specific to quarto, this can be confused with 'overwrite all my folders with empty ones if `initialize_project()` is somehow run twice' which we really don't want to do ever.

Reply:
Updated to `quarto.overwrite` in initialize_project, left as `overwrite` in fetch_quarto_template as I think the scope of overwriting is clear there.